public/js/components/FormFields/FormFieldArrayWrapper.js

```diff
@@ -53,10 +53,10 @@ export default class FormFieldArrayWrapper extends React.Component {
     });
 
     return (
-      <div className={this.props.fieldClass ? this.props.fieldClass : 'form__group'}>
+      <div className={this.props.fieldClass ? this.props.fieldClass : 'form__group form__field'}>
         {this.props.numbered ? <span className="form__field-number">{`${i + 1}. `}</span> : false }
         {hydratedChildren}
-        <button className="btn form__field-btn" type="button" onClick={removeFn.bind(this, i)}>Delete</button>
+        <button className="btn form__field-btn btn--red" type="button" onClick={removeFn.bind(this, i)}>Delete</button>
```

Comment:
I know this _seems_ picky, but `btn--danger` would be more appropriate as a name here.

Reply:
This is using a pre-existing class so not going to change for the moment